_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+>\>
+-----
+
+'>>' is a shell operator that you can use to append the output in an existing file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+$ echo line 2 >> example.txt
+$ cat example.txt
+line 1
+line 2
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+If the file does not exist, '>>' creates the file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+~~~
+
+The appending operator allows for the appending of standard output and standard error to a file.
+
+Appending stdout to the file is appending the output of a file to the end of the other file.
+
+~~~ bash
+$ python random.py >> output.txt
+Traceback (most recent call last):
+  File "random.py", line 2, in <module>
+    print 3 / 0
+ZeroDivisionError: integer division or modulo by zero
+$ cat output.txt
+Random text
+~~~
+
+Appending stderr to the file appends both the stdout and the error message to the end of the other file.
```

Comment:
Hi,

I know that python may not be as used but I'm wondering what is confusing
about using python here. The error message is put on the end of the file of
one and not the other and you can see that here even if you don't usually
use python.

Thanks,
Saul

On Sun, Mar 27, 2016 at 5:00 PM, Mo <notifications@github.com> wrote:

> In _commands/scripting/appendOperator.md
> <https://github.com/c4cs/c4cs.github.io/pull/207#discussion_r57534742>:
>
> > +
> > +The appending operator allows for the appending of standard output and standard error to a file.
> > +
> > +Appending stdout to the file is appending the output of a file to the end of the other file.
> > +
> > +~~~ bash
> > +$ python random.py >> output.txt
> > +Traceback (most recent call last):
> > +  File "random.py", line 2, in <module>
> > +    print 3 / 0
> > +ZeroDivisionError: integer division or modulo by zero
> > +$ cat output.txt
> > +Random text
> > +~~~
> > +
> > +Appending stderr to the file appends both the stdout and the error message to the end of the other file.
>
> This statement isn't entirely correct; that's only because you're
> redirecting stderr to stdout. Like above, this example isn't completely
> clear on what's happening. Please use a different program (perhaps the
> error messages from compiling with gcc?)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/c4cs/c4cs.github.io/pull/207/files/6638ad9f2c8f68956f6028e92574e343f08473bf#r57534742>
>